modules/engine.py

```diff
@@ -22,6 +23,14 @@ def main_loop(self):
                 play = False
             elif command == "help":
                 self.display_help()
+            elif command == "begin":
+                game = modules.world.Room('resources\Game_Setup.csv')
```

Comment:
this variable should probably be self.room rather than game.  self.room is at the object level so it will persist after __init__completes.